src/scvi/model/_multivi.py

```diff
@@ -1117,3 +996,123 @@ def _check_adata_modality_weights(self, adata):
         """
         if (adata is not None) and (self.module.modality_weights == "cell"):
             raise RuntimeError("Held out data not permitted when using per cell weights")
+
+    @classmethod
+    @setup_anndata_dsp.dedent
+    def setup_mudata(
```

Comment:
This should be default. Deprecation warning for anndata should be added and tutorial executed using MuData.

Reply:
OK. will add a deprecation warning for both this and toalvi.
Tutorial wise, maybe we will do it like in totalvi adding the setup_mudata part on top of what already existed.